src/xml/XMLAttribute.hpp

```diff
@@ -155,13 +155,13 @@ void XMLAttribute<ATTRIBUTE_T>::readValue(const std::map<std::string, std::strin
 {
   PRECICE_TRACE(_name);
   if (_read) {
-    PRECICE_ERROR("Attribute \"" + _name + "\" is defined multiple times");
+    PRECICE_ERROR("Attribute \"" + _name + "\" has already been defined. Duplications are not permitted.");
```

Comment:
I cannot trigger this, it is always getting caught earlier, as e.g.: `ERROR: Attribute mesh redefined`

Possible candidate for converting to an assertion.

Reply:
Good point. I agree with you. This should be an assertion.